simple/management/committee_protocol.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+
+import logging, urllib2, re, os
+
+from cStringIO import StringIO
+from django.conf import settings
+from simple.management.utils import antiword
+from pyth.plugins.rtf15.reader import Rtf15Reader
+
+logger = logging.getLogger("open-knesset.syncdata")
+
+DATA_ROOT = getattr(settings, 'DATA_ROOT',
+                    os.path.join(settings.PROJECT_ROOT, os.path.pardir, os.path.pardir, 'data'))
+
+ENCODING = 'utf8'
+
+
+class CommitteProtocolImporter:
```

Comment:
if we are already doing refactoring, then this class should be in committees app